pubsub/oauth.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
```

Comment:
noticed that we have the copyright header in some files and golangci-lint is not complaining, then found [this](https://github.com/conduitio-labs/conduit-connector-salesforce/blob/62afca3a9791a0c28131b0d0f2b7de3cc95e6ee2/.golangci.yml#L14) :D
please uncomment and make sure `golangci-lint` is running successfully